pkg/operator/csi/credentialsrequestcontroller/credentials_request_controller.go

```diff
@@ -40,7 +40,7 @@ func NewCredentialsRequestController(
 	targetNamespace string,
 	manifest []byte,
 	dynamicClient dynamic.Interface,
-	operatorClient v1helpers.OperatorClient,
+	operatorClient v1helpers.OperatorClientWithFinalizers,
```

Comment:
This controller should be removed, actually. In practice, these objects shouldn't be created by any operator expect CVO. This is to support customers running CCO in [Manual Mode](https://github.com/openshift/cloud-credential-operator/blob/master/docs/mode-manual-creds.md).